apis/fluent/atrium-api-fluent/src/commonTest/kotlin/ch/tutteli/atrium/api/fluent/en_GB/CharSequenceToContainAtLeastElementsOfExpectationsTest.kt

```diff
@@ -0,0 +1,149 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import ch.tutteli.atrium.api.verbs.internal.testFactory
+import ch.tutteli.atrium.logic.creating.typeutils.IterableLike
+import ch.tutteli.atrium.specs.emptyIterableLikeTypes
+import ch.tutteli.atrium.specs.integration.AbstractCharSequenceToContainAtLeastExpectationsTest
+import ch.tutteli.atrium.testfactories.TestFactory
+import kotlin.collections.component1
+import kotlin.collections.component2
+
+class CharSequenceToContainAtLeastElementsOfExpectationsTest : AbstractCharSequenceToContainAtLeastExpectationsTest(
+    getAtLeastElementsOfTriple(),
+    getAtLeastIgnoringCaseElementsOfTriple(),
+    getAtLeastButAtMostElementsOfTriple(),
+    getAtLeastButAtMostIgnoringCaseElementsOfTriple(),
+    getNotToContainPair(),
+    getExactlyPair(),
+    Companion::getErrorMsgAtLeastButAtMost
+) {
+
+    companion object : CharSequenceToContainSpecBase() {
```

Comment:
I suggest you add:
```suggestion
    @Test
    fun trigger_run_gutter() = 1
    
    companion object : CharSequenceToContainSpecBase() {
```
This is a workaround so that intellij shows a run gutter for the class. This way you can easier run the tests (e.g. after changing `atLeast(1)` to `atLeast(2)`)